app.go

```diff
@@ -119,6 +119,10 @@ type OIDCConfig struct {
 	Issuer           string
 	ClientID         string
 	ClientSecret     string
+	Scope            []string
+	ExtraParams      map[string]string
+	AllowedDomains   []string
+	AllowedUsers     []string
```

Comment:
This question is rooted in my lack of knowledge of naming conventions for OIDC, is this the appropriate naming? (I dont doubt that it is, but maybe a link to the spec would be nice)